components/CreateNewUserAccount/CreateButton.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Button, Dialog, DialogActions, DialogContent, DialogContentText, DialogTitle } from '@mui/material';
+
+const CreateButton = () => {
+    const [open, setOpen] = React.useState(false);
+
+    const handleClickOpen = () => {
+      setOpen(true);
+    };
+  
+    const handleClose = () => {
+      setOpen(false);
+    };
+  
+    return (
+        <div>
+        <Button variant="contained" onClick={handleClickOpen}>
+          CREATE
+        </Button>
+        <Dialog
+          open={open}
+          onClose={handleClose}
+          aria-labelledby="alert-dialog-title"
+          aria-describedby="alert-dialog-description"
+        >
+          <DialogTitle id="alert-dialog-title" variant="h5">
+            {"Create New User?"}
```

Comment:
nit. you do not need the `{""}`
can just be `<DialogTitle id="alert-dialog-title" variant="h5">Create New User?</DialogTitle>